src/services/README.md

```diff
@@ -0,0 +1,52 @@
+# Services
+Homeday Blocks contains a number of services that we found ourself needing across projects. Some of them are used by Homeday Blocks components, others 
+are here to be consumed in projects directly.
+
+Index:
+- [Breakpoints service](https://github.com/homeday-de/homeday-blocks/tree/develop/src/services#breakpoints) - Used for setting (and getting) the global project breakpoints.
+
+## Breakpoints
+Exposes methods for setting up viewport breakpoints of the project. The breakpoints set in this way are later used by the `HdResponsive` component.
+
+### Usage
+In your project entry file, import the service and define breakpoints that make sense for your project:
+
+```js
+import { setBreakpoints } from 'hd-blocks/services/breakpoints';
+
+const breakpoints = {
+  s: '(max-width:599px)',
+  m: '(min-width:600px)',
+  l: '(min-width:900px)',
+  xl: '(min-width:1200px)',
+};
+setBreakpoints(breakpoints);
+```
+
+You can now import a helper component and render contents based on your breakpoints matching or not:
+
+```vue
+<script>
+import HdResponsive from 'hd-blocks/components/HdResponsive.vue';
+</script>
+
+<template>
+<HdResponsive #default="{ matches, indeterminate }">
+  <div v-if="indeterminate">Matching could not be reliably determined (server side rendering, unit tests environment)</div>
+  <div v-else-if="matches.xl">xl breakpoint matched</div>
+  <div v-else-if="matches.m">m breakpoint matched</div>
+</HdResponsive>
+</template>
+```
+
+As seen in the example above, `HdResponsive` component exposes two arguments:
+- `matches`: Object with the keys matching your breakpoints object and boolean values (`true`: matches, `false`: does not match the current viewport)
```

Comment:
Is worth mentioing breakpoints prop here? I know you did it above already. 

This looks better. :ok_hand:

Reply:
Good point. I mentioned it in the docs.